embc-app/DataInterfaces/DataInterface.Volunteer.cs

```diff
@@ -90,6 +101,13 @@ public async Task<bool> VolunteerExistsAsync(string id)
             return await Volunteers.AnyAsync(x => x.Id == Convert.ToInt32(id));
         }
 
+        public async Task<bool> BceidExistsAsync(string bceid, string volunteerId = null)
+        {
+            int? id = string.IsNullOrEmpty(volunteerId) ? default(int?) : Convert.ToInt32(volunteerId);
+            return await ActiveVolunteers.AnyAsync(x => x.BCeId == bceid &&
+                                            (!id.HasValue || (id.HasValue && x.Id != id)));
```

Comment:
Does this make sense? the check for BCeID uniqueness should be universal, not dependent on the volunteer internal id